components/playlist/browser/media_detector_component_manager.cc

```diff
@@ -74,6 +74,10 @@ base::flat_map<ScriptName, std::string> ReadScriptsFromComponent(
 MediaDetectorComponentManager::MediaDetectorComponentManager(
     component_updater::ComponentUpdateService* component_update_service)
     : component_update_service_(component_update_service) {
+  // TODO(sko) We have breaking changes and not using scripts from component
+  // updater. But we should use script from the component at some point.
+  SetUseLocalScript();
```

Comment:
Since we now know from the component if a `<video>` is `MediaSource`-based, we should no longer need the list we set up in `SetUseLocalListToHideMediaSrcAPI();`, right?

Reply:
`sites_to_hide_media_src_api()` doesn't seem to be used anywhere.

Reply:
> sites_to_hide_media_src_api() doesn't seem to be used anywhere.

Nice catch!